api/src/pcapi/core/users/api.py

```diff
@@ -1520,6 +1531,9 @@ def anonymize_user(user: users_models.User, *, author: users_models.User | None
     for deposit in user.deposits:
         deposit.source = "Anonymized"
 
+    for extract in user.gdprUserDataExtract:
+        delete_gdpr_extract(extract.id)
+
     user.password = b"Anonymized"
```

Comment:
attention git guardian repere cette ligne comme un password il faut le mute (on le fait déjà a d'autres endroits) sinon il va continuer a remonter cette ligne comme un secret hardcodé a chaque fois qu'on travail pas trop loin

Reply:
Pourquoi gitguardian va reprer cette ligne comme un password? 
quel est la différence avec les autres ligne for... in ... delete_gdpr_extract() ? ( par exemple celle contenue ligne 1815 )
- Que veux tu dire par il faut le mute?

Reply:
ok j'ai compris je pensais que c'etait une ligne que j'avais codé mais c'est user.password = b"Anonymized"